PHPfiles/Constants.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * A class to hold Theme related constants.
+ */
+class ThemeConstants {
+    /** Table header background color. */
+    const TableHeaderBGColor = "#D9EDF7";
+    /** Table cell background color when there's no lunch available. */
+    const TableCellNoLunchBGColor = "#D3D3D3";
+    /** Table cell background color when lunch is available to order. */
```

Comment:
Should this say "when lunch is NOT available to order"?

Reply:
Ranamed